omisego/elixir-omg
# feat: init load test app with dummy scenerio

:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Initial dummy `load_test` priv app.

## Changes

- mix new load_test
- add a sample load_test (watcher info) that calls dummy scenario
- add a dummy scenario
- solve conflict with override first. If everyone is happy and we've decide to merge back, will propose a PR to `ethereumex` to upgrade the `httpoison` dependency. See slack discussion: [here](https://omisego.slack.com/archives/C9KNA675J/p1581661226232500?thread_ts=1581497797.143600&cid=C9KNA675J)

## Testing
```
cd priv/
make load_test
```
